server/views/pages/activities/waitlist-application/edit-status.njk

```diff
@@ -0,0 +1,74 @@
+{% extends "layout.njk" %}
+
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "partials/activities/waitlist-status-badge.njk" import waitlistStatusBadge %}
+
+{% set pageTitle = applicationName + " - Waitlist - Status" %}
+{% set pageId = 'status-page' %}
+{% set backLinkHref = '#' %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            <span class="govuk-caption-xl">{{ session.waitListApplicationJourney.activity.activityName }}</span>
+        </div>
+    </div>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <form method="POST">
+                <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+
+                {{ govukRadios({
+                    name: "status",
+                    fieldset: {
+                        legend: {
+                            text: "Change the status of " + session.waitListApplicationJourney.prisoner.name + "'s application",
+                            isPageHeading: true,
+                            classes: "govuk-fieldset__legend--xl"
+                        }
+                    },
+                    hint: {
+                        html: '
+                                <p>If you decline the application, make sure you tell the person concerned.</p>
+                                <div>Select one option</div>
+                              '
+                    },
+                    errorMessage: validationErrors | findError('status'),
+                    items: [
+                        {
+                            value: StatusEnum.APPROVED,
+                            html: waitlistStatusBadgeWithHint(StatusEnum.APPROVED, 'Add the applicant to the waitlist. They’re ready to be allocated.'),
+                            checked: session.waitListApplicationJourney.status == StatusEnum.APPROVED
+                        },
+                        {
+                            value: StatusEnum.DECLINED,
+                            html: waitlistStatusBadgeWithHint(StatusEnum.DECLINED, 'Reject the application and inform the person concerned.'),
```

Comment:
I suspect this may be a content discrepancy. Should be using the word `Decline` in place of `Reject`